ztools/x509/extensions_test.go

```diff
@@ -48,7 +51,7 @@ func (s *ExtensionsSuite) SetUpTest(c *C) {
 
 func (s *ExtensionsSuite) TestEncodeDecodeSAN(c *C) {
 	for _, cert := range s.parsedCerts {
-		if (cert.Issuer.CommonName != "SAN Test"){
+		if cert.Issuer.CommonName != "SAN Test" {
```

Comment:
Zane wrote this. This is just a unit test. He created a cert to go with this unit test that has a CommonName "SAN Test"